scripts/tasks/sandbox.ts

```diff
@@ -143,6 +144,18 @@ export const sandbox: Task = {
       extraDeps,
     });
 
+    if (!options.skipTemplateStories) {
+      for (const addon of options.addon) {
```

Comment:
make sure that options.addon here is the equivalent to addons in the code you moved this from